packages/sdk/src/fee/__tests__/FeeEstimator.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { OrditSDKError } from "../../errors";
+import { FeeEstimator } from "../FeeEstimator";
+import { createMockPsbt } from "../utils";
+
+describe("FeeEstimator", () => {
+  describe("constructor", () => {
+    const INVALID_FEE_RATE_ERROR = new OrditSDKError("Invalid feeRate");
+
+    test("should throw an error for negative fee rate", () => {
+      expect(() => {
+        const _ = new FeeEstimator({
```

Comment:
i think you can drop the const assignment `const _ =` as it proves no value here and subsequent test cases 🤔